README.md

```diff
@@ -106,12 +106,16 @@ A basic use looks like this:
 ```python
 from dagshub.upload import Repo
 
-repo = Repo("idonov8", "baby-yoda-segmentation-dataset", branch="new_annotations")
+# Upload a single file to a repository with one line
+Repo("idonov8", "baby-yoda-segmentation-dataset").upload("file.txt", "commit message") # Optional: versioning, new_branch, last_commit, path
+
+# Upload files to dvc folder in a repository
```

Comment:
I think the point here is "upload multiple files in a directory in a single commit"